### ISSUE TITLE
Create new version of kyma-gke-integration job that uses xip.io

### ISSUE THREAD

**Issue by Tomasz-Smelcerz-SAP** [kyma-project/test-infra#362]
**Description:**
Currently, kyma-gke-integration job allocates DNS/IP resource on every run.
This is because at this moment we don't have support to automate that steps.
Once https://github.com/kyma-project/kyma/issues/1871 is done, we should be able to install and test Kyma without the need to allocate (or setup) any DNS/IP.

**Update:**
Now as xip.io has been merged and became the default method of installation, we can update our PR based pipelines (`kyma-gke-integration`, `upgrade`, `central-connector`) to use xip.io and don't generate DNS/IP on every run. 

This method should speedup the whole pipeline as well as make it less error prone, and resource consuming.

**Comment by Tomasz-Smelcerz-SAP** [comment]
This issue is blocked by:  https://github.com/kyma-project/kyma/issues/1871

**Comment by kasiakepka** [comment]
`pre-master-kyma-xip-integration` and `pre-master-kyma-xip-gke-central-connector` jobs are working as expected however we have issues with `pre-master-kyma-xip-gke-upgrade` job.
It is failing with 

[CreateResources: GrafanaUpgradeTest] End with error [start time: Tue, 23 Apr 2019 11:50:49 UTC, duration: 285.414847ms]: login failed: http request to the url (https://grafana.35.198.142.202.xip.io/login) failed with 'Get https://grafana.35.198.142.202.xip.io/login: dial tcp: lookup grafana.35.198.142.202.xip.io on 10.19.240.10:53: no such host'","taskID":"e3a35c22-216a-4060-af97-a3ce252ef899","time":"2019-04-23T11:50:50.095Z"}}](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/3747/pre-master-kyma-xip-gke-upgrade/1120650893696438272)

https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/3747/pre-master-kyma-xip-gke-upgrade/1120619562799206400

**Comment by kasiakepka** [comment]
After reruning `pre-master-kyma-xip-gke-upgrade` couple times it executed with no errors today, however `pre-master-kyma-xip-gke-central-connector` failed 3/3 times on requests to xip.io domains.
It looks like xip.xio is not that stable, possible it requires more retries or sleeps.

**Comment by kasiakepka** [comment]
I can see that all unsuccessful test failed due to domain/DNS unavailability and require adjustments (more retries or sleeps). While testing XIP integration I noticed two tests that were failing in regular manner:

```
# Test Kyma end-to-end upgrade scenarios & Create e2e upgrade test resources

{"level":"error","log":{"message":"[TestResources: GrafanaUpgradeTest] End with error [start time: Thu, 25 Apr 2019 08:32:30 UTC, duration: 360.350481ms]: login failed: http request to the url (https://grafana.35.246.231.135.xip.io/login) failed with 'Get https://grafana.35.246.231.135.xip.io/login: dial tcp: lookup grafana.35.246.231.135.xip.io on 10.19.240.10:53: no such host'","taskID":"47ebc5d5-7992-44d1-870a-95b42a72a943","time":"2019-04-25T08:32:30.384Z"}}

```
and
```
Testing 'oct-tp-testsuite-all-2019-04-24-07-45-test-core-core-console-backend-service-0' from namespace 'kyma-system'
'oct-tp-testsuite-all-2019-04-24-07-45-test-core-core-console-backend-service-0' has Failed status

Fetching logs from 'oct-tp-testsuite-all-2019-04-24-07-45-test-core-core-console-backend-service-0' with options tailLimit=2000 and bytesLimit=500000

Post https://dex.35.198.182.87.xip.io/auth: dial tcp: lookup dex.35.198.182.87.xip.io on 10.59.240.10:53: no such host

```

**Comment by kasiakepka** [comment]
After discussion with Szymon the easiest  and quickest option would be to add WrappedHttpClient that will handle retries for test that are failing.

**Comment by kasiakepka** [comment]
Currently upgrade pipeline is failing on `e2e-upgrade-test`

{"level":"error","log":{"message":"wrong response: test-event-31, want: test-event-96","taskID":"f51162a6-317d-438e-9d44-2a19285ed010","time":"2019-06-13T12:21:43.808Z"}}
{"level":"error","log":{"message":"TestResources() failed with: timeout for subscriber response","taskID":"f51162a6-317d-438e-9d44-2a19285ed010","time":"2019-06-13T12:21:43.808Z"}}
{"level":"error","log":{"message":"[TestResources: EventBusUpgradeTest] End with error [start time: Thu, 13 Jun 2019 12:18:33 UTC, duration: 3m10.180870963s]: timeout for subscriber response","taskID":"f51162a6-317d-438e-9d44-2a19285ed010","time":"2019-06-13T12:21:43.808Z"}}
https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_kyma/4530/pre-master-kyma-xip-gke-upgrade/1139136120710238209

gke-integration and gke-central-connector are working with no issues

**Comment by kasiakepka** [comment]
This issue is tracked here: https://github.com/kyma-project/test-infra/issues/1206

**Comment by adamwalach** [comment]
It's no longer needed. XIP is not used any more in our pipelines, and we have no plans to bring it back.
